test/regtest.py

```diff
@@ -75,11 +83,11 @@
   RegTest('array3_fail',           r'0 verified, 1 errors?', r'This assertion can fail', r'This assertion can fail', 11),
   RegTest('array4',                r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 11),
   RegTest('array4_fail',           r'0 verified, 1 errors?', r'This assertion can fail', r'This assertion can fail', 11),
-#  RegTest('array_free',            r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 11),
-#  RegTest('array_free_fail',       r'0 verified, 3 errors?', r'This assertion can fail', r'This assertion can fail', 11),
-#  RegTest('array_free1',           r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 11),
-#  RegTest('array_free1_fail',      r'0 verified, 4 errors?', r'This assertion can fail', r'This assertion can fail', 11),
-#  RegTest('array_free2',           r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 11),
+  RegTest('array_free',            r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 11),
+  RegTest('array_free_fail',       r'0 verified, 3 errors?', r'This assertion can fail', r'This assertion can fail', 11),
+  RegTest('array_free1',           r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 11),
+  #RegTest('array_free1_fail',      r'0 verified, 4 errors?', r'This assertion can fail', r'This assertion can fail', 11),
```

Comment:
Since there is `--bit-vector` field for this script, I think we should split the list into two.